### PR TITLE
Feature metadata loader

### PR DESCRIPTION
**Pull request by lilleyse**
Follow up to https://github.com/CesiumGS/cesium/pull/9481

Adds a loader for the `EXT_feature_metadata` extension. `GltfFeatureMetadataLoader` loads textures, buffer views, and schemas from the cache but does not itself need to go in the cache, so there are no changes to `ResourceCache` and `ResourceCacheKey`.

114327478-34551680-9b07-11eb-9d65-c71e43f177b3

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Specs/Scene/GltfFeatureMetadataLoaderSpec.js [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+          expect(featureMetadataLoader.featureMetadata).not.toBeDefined();
+          expect(featureMetadataLoader.isDestroyed()).toBe(true);
+
+          expect(destroyBufferView.calls.count()).toBe(6);
+          expect(destroyTexture.calls.count()).toBe(2);
+          expect(destroySchema.calls.count()).toBe(1);
+        });
+      });
+    });
+  },
```
Comment (ptrgags) [inline Specs/Scene/GltfFeatureMetadataLoaderSpec.js:483]: looking at the coverage, one thing you could test is that it handles the case where the object is destroyed before load finishes, like we did with the other loaders.

**Review by ptrgags — COMMENTED** [review]
@lilleyse looks good overall, just noticed one unit test that could be added to make sure a corner case is covered.

**Comment by lilleyse** [comment]
@ptrgags I added the additional test. Should be ready now.

**Comment by ptrgags** [comment]
looks good now, thanks @lilleyse!
